bin/start-vmagent

```diff
@@ -34,6 +34,7 @@ erb config/vmagent-prometheus.yml.erb > config/vmagent-prometheus.yml
   echo -n "${VMAGENT_REMOTE_WRITE_PASSWORD}" > /tmp/vmagent-password
   bin/vmagent \
     -promscrape.config=config/vmagent-prometheus.yml \
+    -memory.allowedBytes="${VMAGENT_ALLOWED_BYTES:-104857600}" \
```

Comment:
Let's document this variable [here](https://github.com/ably/heroku-buildpack-vmagent#configuration)